usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -94,15 +94,15 @@ LEFT JOIN (
 LEFT JOIN (
   SELECT
     faba.award_id,
-    JSONB_AGG(
-      DISTINCT JSONB_BUILD_OBJECT(
```

Comment:
The distinct was important before. Can you confirm the new SQL doesn't contain duplicate TAS in the array?

Reply:
The new SQL did produce duplicates, fixed with latest commit